ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -113,6 +113,47 @@ nnfw_get_accelerator (nnfw_pdata * pdata, const char *accelerators)
   }
 }
 
+/**
+ * @brief Parse accelerator and other option.
+ */
+static void
+nnfw_parse_custom_option (const GstTensorFilterProperties * prop,
+    nnfw_pdata * pdata)
+{
+  if (prop->custom_properties) {
+    gchar **options;
+    guint i, len;
+
+    options = g_strsplit (prop->custom_properties, ",", -1);
+    len = g_strv_length (options);
+
+    for (i = 0; i < len; ++i) {
+      gchar **option = g_strsplit (options[i], ":", -1);
+
+      if (g_strv_length (option) > 1) {
+        g_strstrip (option[0]);
+        g_strstrip (option[1]);
+
+        if (g_ascii_strcasecmp (option[0], "Runtime") == 0) {
+          pdata->accelerator = g_strdup (option[1]);
```

Comment:
Q. Is there any possibility for `pdata->accelerator` to be overwritten in this for loop?

Reply:
currently no, nnfw_parse_custom_option() is always called with newly allocated nnfw_pdata ptr.